src/rest-server/src/config/v2/user.js

```diff
@@ -74,7 +74,7 @@ const userCreateInputSchema = Joi.object().keys({
     .required(),
   extension: Joi.object()
     .pattern(/\w+/, Joi.required())
-    .default({}),
```

Comment:
Keep it

Reply:
For object, it's better to keep a empty default, see https://github.com/hapijs/joi/issues/1135

Reply:
Can you test this API and request it without the extension field. Could share me with the result?

Reply:
Create user:
![image](https://user-images.githubusercontent.com/11887940/62995583-c53ba080-be93-11e9-8cfc-107ae656ea38.png)

Get user:
![image](https://user-images.githubusercontent.com/11887940/62995602-d684ad00-be93-11e9-8fae-80a6af42cf22.png)

Reply:
For object, we should use `default()` instead of `default({})`, they could achive the same result, using `default()` is less possible to make mistakes.